### PR TITLE
[FIX] Fixed URL issue

### PR DESCRIPTION
**Pull request by AadilRayeen**
- Fixed the error in the URL, mentioned on 8 line of About page of TCET Linux Website Docs from (https://tcet-linux-website/) to (https://linux.tcetmumbai.in/).

### CODE REVIEW

File: docs/projects/tcet-linux-website/about-tcet-linux-website.mdx [hunk 1]
```diff
@@ -5,7 +5,7 @@ sidebar_label: TCET Linux Website
 description: Official documentation of TCET Linux Website 
 ---
 
->_This is the official documentation of the **[TCET Linux website](https://tcet-linux-website/)** cell of **[TCET Mumbai](https://tcetmumbai.in)**, maintained by the **[TCET Open Source](https://opensource.tcetmumbai.in)** Documentation team._
+>_This is the official documentation of the **[TCET Linux website](https://linux.tcetmumbai.in/)** cell of **[TCET Mumbai](https://tcetmumbai.in)**, maintained by the **[TCET Open Source](https://opensource.tcetmumbai.in)** Documentation team._
```
Comment (vish-rt) [inline docs/projects/tcet-linux-website/about-tcet-linux-website.mdx:8]: I believe it is redundant to copy the description of T&P Website for this project as both projects are created for different purposes. Here, we have used the word `cell` which is irrelevant to TCET Linux Website.
```suggestion
>_This is the official documentation of the **[TCET Linux Website](https://linux.tcetmumbai.in/)** for **TCET Linux**: a GNU/Linux distribution, maintained by the **[TCET Open Source](https://opensource.tcetmumbai.in)** Documentation team._
```

You can even mention the [Teams](https://github.com/orgs/tcet-opensource/teams/devops-team) that have built the project.

P.S: I'm aware this is beyond the scope of the issue, but adding these changes can be fruitful for the overall documentation.

**Review by vish-rt — CHANGES_REQUESTED** [review]

**Comment by AadilRayeen** [comment]
Changes made as per required, kindly review!!!!!!!!!!!!!

**Review by vish-rt — APPROVED** [review]
Looks good to me

**Comment by Akash6222** [comment]
2023-09-02T12:50:09,313881366+05:30

This feel weird to Get Started with TCET Linux Website, I think it should be Get Started with TCET Linux and link should get to TCET Linux Website

**Comment by himanshu-03** [comment]
> 2023-09-02T12:50:09,313881366+05:30
> 
> This feel weird to Get Started with TCET Linux Website, I think it should be Get Started with TCET Linux and link should get to TCET Linux Website

Ig its because documentation of TCET Linux website and TCET Linux OS is different.

**Comment by vish-rt** [comment]
I believe that's beyond the scope of this issue. We should definitely circle back around it when the team has enough bandwidth to address it.

**Review by himanshu-03 — APPROVED** [review]
LGTM! 
Can ship to main 🚀

**Pull request merged**